perfkitbenchmarker/benchmark_spec.py

```diff
@@ -338,14 +338,23 @@ def PrepareVm(self, vm):
     Args:
         vm: The BaseVirtualMachine object representing the VM.
     """
+    vm_metadata = {'benchmark': self.name,
+                   'perfkit_uuid': self.uuid,
+                   'benchmark_uid': self.uid}
+    if FLAGS.vm_metadata:
+      for item in FLAGS.vm_metadata:
+        if ':' not in item:
+          raise Exception('\'%s\' not in expected key=value format' % item)
```

Comment:
in order to avoid escaping quotes you can either use double quotes on the string or use double quotes on "%s" -  this isn't that bad though so only fix if you feel like it